web/viewer.css

```
@@ -487,14 +489,52 @@ select {
   background-color: rgba(255, 102, 102, 1);
 }
 
-.secondaryToolbar {
+.secondaryToolbar,
+.editorParamsToolbar {
   padding: 6px 0 10px;
   inset-inline-end: 4px;
   height: auto;
   z-index: 30000;
   background-color: var(--doorhanger-bg-color);
 }
 
+#editorParamsToolbarContainer > .editorParamsSetter {
+  max-width: 220px;
+  min-height: 26px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-left: 12px;
+  padding-right: 12px;
```

Please always use *logical* CSS properties in the UI, to avoid any future issues:
```suggestion
  padding-inline-start: 12px;
  padding-inline-end: 12px;
```